python/pyarrow/parquet.py

```diff
@@ -1181,6 +1181,11 @@ def _make_manifest(path_or_paths, fs, pathsep='/', metadata_nthreads=1,
     If the source is a file path, use a memory map to read file, which can
     improve performance in some environments
 {1}
+filters : List[Tuple] or List[List[Tuple]] or None (default)
```

Comment:
This will appear in the `read_pandas` docstring, so you should probably add the `filters` argument to `read_pandas` as well.